### PR TITLE
LPS-75815

### PR DESCRIPTION
**Pull request by leoadb**

### CODE REVIEW

**Comment by inacionery** [comment]
ci:test

File: dynamic-data-mapping-form-analytics/src/main/java/com/liferay/dynamic/data/mapping/form/analytics/internal/servlet/taglib/DDMFormAnalyticsDynamicInclude.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+@Component(immediate = true, service = DynamicInclude.class)
+public class DDMFormAnalyticsDynamicInclude extends BaseJSPDynamicInclude {
+
+	@Override
+	public void include(
+			HttpServletRequest request, HttpServletResponse response,
+			String key)
+		throws IOException {
+
+		ScriptData scriptData = new ScriptData();
```
Comment (inacionery) [inline dynamic-data-mapping-form-analytics/src/main/java/com/liferay/dynamic/data/mapping/form/analytics/internal/servlet/taglib/DDMFormAnalyticsDynamicInclude.java]: @leoadb puts this variable next to its first use. Line 53.

File: dynamic-data-mapping-form-analytics/src/main/java/com/liferay/dynamic/data/mapping/form/analytics/internal/servlet/taglib/DDMFormAnalyticsDynamicInclude.java [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+	@Override
+	@Reference(
+		target = "(osgi.web.symbolicname=com.liferay.dynamic.data.mapping.form.analytics)",
+		unbind = "-"
+	)
+	protected void setServletContext(ServletContext servletContext) {
+		super.setServletContext(servletContext);
+	}
+
+	private static final String _ANALYTICS_GATEWAY_URL = System.getProperty(
```
Comment (inacionery) [inline dynamic-data-mapping-form-analytics/src/main/java/com/liferay/dynamic/data/mapping/form/analytics/internal/servlet/taglib/DDMFormAnalyticsDynamicInclude.java]: Hey @leoadb, this should look like this https://github.com/liferay/com-liferay-analytics/blob/master/analytics-client-osgi/src/main/java/com/liferay/analytics/client/osgi/internal/AnalyticsClientImpl.java#L89

**Comment by pedroqueiroz94** [comment]
guys, we're waiting on Shuyang's team to have this fixed. :( cc. 

@rodrigo-e-c-souza @vicnate5

**Pull request merged**